app/content/drill/[id]/leaderboard.js

```diff
@@ -95,49 +111,25 @@ export default function Leaderboard() {
     : customMainOutputAttempt;
 
   const leaderboardAttempts = preCalcLeaderboard || {};
-  if (!preCalcLeaderboardExists && attempts) {
-    //just in case...
-    for (const id in attempts) {
-      const entry = attempts[id];
-
-      console.log("ENTRYID", entry.id);
-
-      const lowerIsBetter =
-        drillInfo["aggOutputs"][mainOutputAttempt]["lowerIsBetter"];
-      // If this uid has not been seen before or the current score is higher, store it
-      if (
-        !leaderboardAttempts[entry.uid] ||
-        (lowerIsBetter &&
-          leaderboardAttempts[entry.uid][mainOutputAttempt]["value"] <
-            entry[mainOutputAttempt]) ||
-        (!lowerIsBetter &&
-          leaderboardAttempts[entry.uid][mainOutputAttempt]["value"] >
-            entry[mainOutputAttempt])
-      ) {
-        leaderboardAttempts[entry.uid] = {
-          [mainOutputAttempt]: {
-            value: entry[mainOutputAttempt],
-            id: entry.id,
-          },
-        };
-      }
-    }
-
-    updateLeaderboard({
-      currentTeamId,
-      drillId,
-      value: leaderboardAttempts,
-    });
-  }
 
   // console.log("drillLeaderboardAttempts: ", leaderboardAttempts);
-
+  const sortByLower = drillInfo["aggOutputs"][mainOutputAttempt][
+    "lowerIsBetter"
+  ]
+    ? 1
+    : -1;
   const orderedLeaderboard = Object.keys(leaderboardAttempts).sort(
     //only sort the userId
     (a, b) =>
-      leaderboardAttempts[a][mainOutputAttempt]["value"] -
-      leaderboardAttempts[b][mainOutputAttempt]["value"],
+      sortByLower * leaderboardAttempts[a][mainOutputAttempt]["value"] +
```

Comment:
sortByLower * (leaderboardAttempts[a][mainOutputAttempt]["value"] - leaderboardAttempts[b][mainOutputAttempt]["value"]) ?

Reply:
yeah I just wrote it the weird way cause it was easier to think about math wise, but I can change this for code legibility lol